test/test_case_fixture.h

```diff
@@ -374,10 +374,14 @@ struct test_case_fixture : public base_test_fixture
             REQUIRE(columns.column_name() == NANODBC_TEXT("c3"));
             if (vendor_ == database_vendor::sqlite)
             {
-#ifdef _WIN32
+                // FIXME: SQLite ODBC mis-reports decimal digits? Causing
+                //        columns.column_size() == 3.
+#if defined _WIN32
```

Comment:
Minor, style: `#if defined(_WIN32)` would be consistent with currently used preprocessor style.